lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -365,6 +368,9 @@ def __post_init__(self, *args: Any, **kwargs: Any) -> None:
                     "should be specified either both or not."
                 )
 
+        if self.use_bicep is True:
```

Comment:
It doesn't need to handle this flag here. Just use it in `_load_template` method.

Reply:
done